src/lib/y2partitioner/widgets/encrypt_method_options.rb

```diff
@@ -54,10 +54,11 @@ def empty_widget
       # @return [CWM::CustomWidget] a widget containing the options for the
       # given encryption method
       def options_for(encrypt_method)
```

Comment:
I was tempted to use a guard clause here, 


```ruby
def options_for(encrypt_method)
  return RandomOptions.new(controller) if encryp_method.is?(:random_swap)

  LuksOptions.new(controller)
end
```
But then I decided to go ahead with a `case` statement because of the upcoming _random swap_ encryption methods.